Internal flaky test: expected 'assert' to equal 'invoke'

### Current behavior:

I've seen this fail sometimes in firefox. 

**CircleCI Failed Run:** https://circleci.com/gh/cypress-io/cypress/335490#tests/containers/1
**Failing Spec:** https://github.com/cypress-io/cypress/blob/develop/packages/driver/test/cypress/integration/commands/connectors_spec.js#L367:L367

<img width="1521" alt="Screen Shot 2020-05-27 at 2 10 47 PM" src="https://user-images.githubusercontent.com/1271364/82991703-16341400-a024-11ea-8ea4-60187b934464.png">

```
  1) src/cy/commands/connectors
       with jquery
         #invoke
           assertion verification
             eventually fails the assertion:

      AssertionError: expected 'invoke' to equal 'assert'
      + expected - actual

      -'invoke'
      +'assert'
      
  ./cypress/integration/commands/connectors_spec.js/</</</</</<@http://localhost:3500/__cypress/tests?p=cypress/integration/commands/connectors_spec.js:424:44
  listener@cypress:///../driver/src/cypress/events.js:105:19
  arrayMap@cypress:////root/cypress/node_modules/lodash/lodash.js:639:23
  map@cypress:////root/cypress/node_modules/lodash/lodash.js:9554:14
  extend/events.emitMap@cypress:///../driver/src/cypress/events.js:109:16
  _loop/parent[fn]@cypress:///../driver/src/cypress/events.js:55:34
  action@cypress:///../driver/src/cypress.js:477:31
  fail@cypress:///../driver/src/cypress/cy.js:739:22
  run/promise<@cypress:///../driver/src/cypress/cy.js:576:14
  tryCatcher@cypress:////root/cypress/node_modules/bluebird/js/release/util.js:16:23
  module.exports/Promise.prototype._settlePromiseFromHandler@cypress:////root/cypress/node_modules/bluebird/js/release/promise.js:512:31
  module.exports/Promise.prototype._settlePromise@cypress:////root/cypress/node_modules/bluebird/js/release/promise.js:569:18
  module.exports/Promise.prototype._settlePromise0@cypress:////root/cypress/node_modules/bluebird/js/release/promise.js:614:10
  module.exports/Promise.prototype._settlePromises@cypress:////root/cypress/node_modules/bluebird/js/release/promise.js:690:18
  _drainQueueStep@cypress:////root/cypress/node_modules/bluebird/js/release/async.js:138:12
  _drainQueue@cypress:////root/cypress/node_modules/bluebird/js/release/async.js:131:24
  Async.prototype._drainQueues@cypress:////root/cypress/node_modules/bluebird/js/release/async.js:147:16
  Async/this.drainQueues@cypress:////root/cypress/node_modules/bluebird/js/release/async.js:17:14
```

### Versions

4.7.0

--- a/packages/driver/test/cypress/integration/commands/connectors_spec.js
+++ b/packages/driver/test/cypress/integration/commands/connectors_spec.js
@@ -335,7 +335,7 @@ describe('src/cy/commands/connectors', () => {
         beforeEach(function () {
           delete this.remoteWindow.$.fn.foo
 
-          Cypress.config('defaultCommandTimeout', 100)
+          Cypress.config('defaultCommandTimeout', 200)
 
           this.logs = []
 

